spec/fixtures/hiera/hathitrust.yaml

```diff
@@ -86,3 +86,15 @@ nebula::hathitrust::monitor::solr_cores:
 nebula::hathitrust::monitor::mysql:
   param1: value1
   param2: value2
+
+nebula::profile::hathitrust::ingest_service::config: '/default/feedd.yaml'
+nebula::profile::hathitrust::ingest_jobs::config: '/default/feedd.yaml'
+nebula::profile::hathitrust::ingest_jobs::rights_db_config: '/default/rights_db.yaml'
+nebula::profile::hathitrust::ingest_jobs::feed_home: '/htfeed'
+nebula::profile::hathitrust::ingest_jobs::recipient: 'nobody@default.invalid'
+nebula::profile::hathitrust::ingest_jobs::dcu_recipients:
+  - aardvark@default.invalid
+  - badger@default.invalid
+
+nebula::profile::hathitrust::ingest_jobs::crms_renewals_source: libadm@latte-1:/l1/prep/c/crms/CRMSRenewals.tsv
```

Comment:
Mock this in test?

Reply:
This is the test mock.

Reply:
Or do you mean _not_ have it in hiera for the test and just provide it via `:params`?